src/Options.cc

```diff
@@ -400,15 +403,15 @@ Options parse_cmdline(int argc, char** argv)
 		{"mem-profile", no_argument, nullptr, 'M'},
 #endif
 
-		{"pseudo-realtime", optional_argument, nullptr, 'E'},
+		{"pseudo-realtime", optional_argument, nullptr, '~'},
```

Comment:
`E` was supported previously but not actually documented?

Reply:
yeah - hard to imagine that repurposing it will cause headaches, given it being undocumented, and pseudo-realtime only being of interest to connoisseurs